WcaOnRails/app/models/regional_organization.rb

```diff
@@ -30,4 +30,13 @@ def start_date_must_be_earlier_than_end_date
   def is_pending?
     start_date.nil?
   end
+
+  def serializable_hash(options = nil)
+    {
+      name: name,
+      website: website,
+      country: country,
+      logo: logo.attached? ? Rails.application.routes.url_helpers.rails_representation_url(logo.variant(resize: "500x300").processed) : '',
```

Comment:
Semantically it's better to use `nil` for absent fields :)

```suggestion
      logo: logo.attached? ? Rails.application.routes.url_helpers.rails_representation_url(logo.variant(resize: "500x300").processed) : nil,
```